src/service/action-impl.js

```diff
@@ -492,7 +496,8 @@ export class ActionService {
     let n = target;
     while (n) {
       const actionInfos = this.matchActionInfos_(n, actionEventType);
-      if (actionInfos) {
+      const isEnabled = !matches(n, DISABLED_PSEUDOCLASS_);
```

Comment:
I see. That's crappy. But would it make more to do `closest(':disabled')` for X where `X.disabled !== undefined`